src/lib/utils/formatNumber.js

```diff
@@ -0,0 +1,10 @@
+import numeral from 'numeral'
+
+// use this function to format the number to be with the abbreviations (i.e. 100 => 1K, 10000 => 10K, 2000000 => 2M, 20000000 => 20M)
+export const formatNumberToBeWithAbbreviations = (number, customFormat) => {
```

Comment:
let's rename it to `formatWithSIPrefix` or just `formatSIPrefix`
those "abbreviations" are metric prefixes defined by International System of Units